docs/Reference/JSON-RPC-API-Objects.md

```diff
@@ -0,0 +1,155 @@
+| `parentHash` | *Data*, 32 bytes | Hash of the parent block.| 
+| `nonce` | *Data*, 8 bytes | Hash of the generated proof of work. `null` when block is pending. | 
+|  `sha3Uncles` | *Data*, 32 bytes | SHA3 of the uncle's data in the block. | 
+| `logsBloom` | *Data*, 256 bytes | Bloom filter for the block logs. `null` when block is pending.|
+| `transactionsRoot` | *Data*, 32 bytes | Root of the transaction trie for the block. |
+| `stateRoot` | Data, 32 bytes | Root of the final state trie for the block. | 
+| `receiptsRoot` | Data, 32 bytes | Root of the receipts trie for the block. | 
+| `miner` | Data, 20 bytes |  Address to which mining rewards were paid. | 
+| `difficulty` | Quantity, Integer | Difficulty for this block. |
+| `totalDifficulty` | Quantity, Integer | Total difficulty of the chain until this block.|
+| `extraData` | Data | Extra data field of this block. | 
+| `size` | Quantity, Integer | Size of block in bytes. | 
+| `gasLimit` |  Quantity | Maximum gas allowed in this block. | 
+| `gasUsed` | Quantity | Total gas used by all transactions in this block. |
+| `timestamp` | Quantity | Unix timestamp when block was collated. |
+| `transactions` | Array | Array of [transaction objects](#transaction-object), or 32 byte transaction hashes depending on the specified boolean parameter.|
+| `uncles` | Array | Array of uncle hashes.| 
+
+## Filter Options Object
+
+Parameter for [eth_newFilter](JSON-RPC-API-Methods.md#eth_newfilter) and [eth_getLogs](JSON-RPC-API-Methods.md#eth_getlogs). 
+
+| Key   |   Type            | Required/Optional | Value  |  
+|-------|:-----------------:|:-----------------:|------|
+| `fromBlock`| Quantity &#124;Tag | Optional | Integer block number or `latest`, `pending`, `earliest`. See [Block Parameter](Using-JSON-RPC-API.md#block-parameter). Default is `latest`.|
+| `toBlock` | Quantity &#124;Tag | Optional | Integer block number or `latest`, `pending`, `earliest`. See [Block Parameter](Using-JSON-RPC-API.md#block-parameter). Default is `latest`.|
+| `address`| Data &#124; Array | Optional | Contract address or array of addresses from which logs originate.|
+| `topics` | Array of Data, 32 bytes each | Optional| Array of 32-byte topics. Topics are order-dependent. Each topic can also be an array of DATA with "or" options. |
+
+[eth_getLogs](JSON-RPC-API-Methods.md#eth_getlogs) has an additional key. 
+
+| Key   |   Type            | Required/Optional | Value  |  
+|------------|:-----------------:|:-----------------:|------|
+| `blockhash`|Data, 32 bytes | Optional | Hash of block for which to return logs. If `blockhash` is specified, `fromBlock` and `toBlock` cannot be specified. | 
+
+## Log Object 
+
+Returned by [eth_getFilterChanges](JSON-RPC-API-Methods.md#eth_getfilterchanges) and [transaction receipt objects](#transaction-receipt-object) can contain an array of log objects.  
+
+| Key   |   Type            |  Value  |  
```

Comment:
Is it possible to make the left hand column of these object tables wider?  The key are currently wrapping but the right column is really wide.  Not sure how much control we have over md tables though.

Reply:
It's due to the abusive use of code highlighting to emphasise words. Bold would be more effective in this context and the word wrap would be correct. Also try to format the tables so that they are readable in the markdown code.